docs/root/api-v1/tracing.rst

```diff
@@ -55,7 +55,8 @@ Zipkin driver
     "type": "zipkin",
     "config": {
       "collector_cluster": "...",
-      "collector_endpoint": "..."
+      "collector_endpoint": "...",
+      "trace_id_128bit": true
```

Comment:
Let's not add any new v1 config options. For v1 configs just hard code it to off during the translation? Do you mind unwinding these changes and the schema changes?

Reply:
Updated - let me know if I have missed anything or misunderstood.